lib/jets/controller/params.rb

```diff
@@ -82,12 +82,14 @@ def body_params
     alias_method :body_parameters, :body_params
 
   private
-
     def parse_multipart(body)
       boundary = ::Rack::Multipart::Parser.parse_boundary(headers["content-type"])
       options = multipart_options(body, boundary)
       env = ::Rack::MockRequest.env_for("/", options)
-      ::Rack::Multipart.parse_multipart(env) # params Hash
+
+      result = ::Rack::Multipart.parse_multipart(env) # params Hash
+      result['file'].transform_values! { |v| v.instance_of?(String) ? v.force_encoding('UTF-8') : v }
```

Comment:
pay attention, reductions are still not acceptable

Reply:
fixed

Reply:
it is already approved in any case..

Reply:
so before merging it (and b2b PR) we should check it on test ENV

Reply:
checked, it works